src/uct/ib/rc/base/rc_iface.c

```diff
@@ -898,3 +893,29 @@ ucs_status_t uct_rc_iface_fence(uct_iface_h tl_iface, unsigned flags)
     return UCS_OK;
 }
 
+ucs_status_t uct_rc_iface_init_pending_mp(uct_rc_iface_t *iface)
+{
+    ucs_status_t status;
+
+    if (iface->tx.pending_mp_initialized != 0) {
```

Comment:
minor: can use just `if (iface->tx.pending_mp_initialized)`, because it is "boolean"

Reply:
ok, fixed